### ISSUE TITLE
Less ram?

### ISSUE THREAD

**Issue by oegjoe** [windowtoolbox/powershell-windows-toolbox#1]
Is there a way that the windows subsystem is able to work with less ram? or any other work arounds?

**Comment by windowtoolbox** [comment]
NO, windows subsystem for android required minimum of 8 gbs of ram. below 8 gbs ram, WSA is not usable as it will crash.

**Comment by Janmm14** [comment]
Malware - Especially english windows installs at risk, see #32
